src/celeritas/io/ImportPhysicsVector.hh

```diff
@@ -46,6 +46,27 @@ struct ImportPhysicsVector
     }
 };
 
+//---------------------------------------------------------------------------//
+/*!
+ * Store imported 2D physics vector data (see Geant4's G4Physics2DVector.hh).
+ *
+ * This stores a 2D grid of generic data with linear interpolation.
+ */
+struct ImportPhysics2DVector
```

Comment:
Can you also replace the `ImportSBTable` with this structure? It's got the same data+layout.